tests/deepspeed/test_deepspeed.py

```diff
@@ -643,7 +643,7 @@ def test_early_get_last_lr(self, stage, dtype):
             # print(trainer.model.b.item())
             # need to investigate at some point
             if (stage == ZERO3 and dtype == FP16) or (dtype == BF16):
-                return
+                self.skipTest("When using zero3/fp16 or any/bf16 the optimizer seems run oddly")
```

Comment:
should we want to add `reason=` here too (and other similar places)?

Reply:
Sure! I can add that